rest_log/components/service.py

```diff
@@ -93,11 +79,17 @@ def _dispatch_exception(
             log_entry_url = self._get_log_entry_url(log_entry)
         # UserError and alike have `name` attribute to store the msg
         exc_msg = self._get_exception_message(orig_exception)
-        raise exception_klass(exc_msg, log_entry_url) from orig_exception
+        exc = exception_klass(exc_msg, log_entry_url)
+        # Retrieve REST JSON info from original exception (if existing)
+        exc.rest_json_info.update(self._get_rest_json_info(orig_exception))
+        raise exc from orig_exception
```

Comment:
```suggestion
        json_info = {"log_entry_url": log_entry_url}
        # Retrieve REST JSON info from original exception (if existing)
        json_info.update(self._get_exception_json_info(orig_exception)))
        exc = exception_klass(exc_msg, **json_info)
        raise exc from orig_exception
```
and modify the `__init__` of the exception to smth like:

 ```
    def __init__(self, message, log_entry_url=None, **kw):
        super().__init__(message)
        self.rest_json_info = {"log_entry_url": log_entry_url, **kw}
``